tests/test_agents/test_apy_estimation_abci.py

```diff
@@ -68,6 +68,10 @@ class BaseTestABCIAPYEstimationSkillNormalExecution(BaseTestEnd2EndExecution):
             "value": 200,
             "type_": "float",
         },
+        {
+            "dotted_path": f"{__args_prefix}.optimizer.timeout",
+            "value": 1,
+        },
```

Comment:
If the first trial requires more time than the round's timeout, then the study will never be interrupted, and the round will timeout. This is because `optuna` checks for the timeout after every trial finishes, and therefore it is possible for it to be exceeded. 

Therefore, we increase the round's timeout and also decrease the optimizer's timeout for the e2e tests.